app/views/drivers/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<section class="container mt-3">
+  <h2>List of Drivers</h2>
+  <nav><%= link_to "Create New Driver", new_driver_path, class:"btn btn-info m-3" %></nav>
+  
+  <% if @drivers.length > 0 %>
```

Comment:
Consider how you could make use of a partial view for the drivers and passengers index page.